randovania/games/fusion/logic_database/Sector 1 SRX.txt

```diff
@@ -1244,12 +1249,14 @@ Extra - room_id: [35]
   > Door to Checkpoint Intersection
       Any of the following:
           # Deal with the Pirates
-          Charge Beam or Missiles ≥ 10 or Combat (Intermediate) or Can Freeze Enemies or Can Use Power Bombs
+          Charge Beam or Missiles ≥ 10 or Can Kill Super Geron or Can Use Power Bombs
```

Comment:
this should get changed back to `Can Freeze Enemies`

Reply:
Odd, I don't recall changing this. Perhaps I accidentally scrolled over the field. When looking through the logic